packages/framework/fluid-framework/api-report/fluid-framework.api.md

```diff
@@ -11,7 +11,7 @@ import { ContainerSchema } from '@fluidframework/fluid-static';
 import { DataObjectClass } from '@fluidframework/fluid-static';
```

Comment:
'fluid-framework.api.md' re-exports * from 'fluidframework/tree', so same diffs appear here.

(Soon, it will only re-export the public the public rollup.)